packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -0,0 +1,155 @@
+import PropTypes                from 'prop-types';
+import React                    from 'react';
+import {
+    Money,
+    Icon,
+    ThemedScrollbars }          from '@deriv/components';
+import { localize }             from '@deriv/translations';
+import {
+    epochToMoment,
+    toGMTFormat }               from 'Utils/Date';
+import {
+    getBarrierLabel,
+    getBarrierValue,
+    isDigitType }               from 'App/Components/Elements/PositionsDrawer/helpers';
+import {
+    isDealCancellationExpired,
+    isUserCancelled,
+    isUserSold }                from 'Stores/Modules/Contract/Helpers/logic';
+import { isMultiplierContract } from 'Stores/Modules/Contract/Helpers/multiplier';
+import ContractAuditItem        from './contract-audit-item.jsx';
+
+const ContractDetails = ({
+    contract_end_time,
+    contract_info,
```

Comment:
Tracking object changes in props is a bit hard, can we extract the required props from the `contract_info` in the props? I don't mean changing the functionality, but something like: 
```jsx
({
contract_info: { transaction_ids, contract_type, commision, currency, deal_cancellation, etc... }
})
```

Reply:
sure 👍